dev-client/src/screens/SoilScreen/ColorScreen/utils/munsellConversions.ts

```diff
@@ -213,3 +226,27 @@ export const munsellToString = (color: MunsellColor) => {
 
   return `${hueSubstep}${hue} ${value}/${chroma}`;
 };
+
+export const nearestSoilColor = (color: MunsellHVC) =>
+  FLATTENED_SOIL_COLORS.reduce((a, b) =>
+    munsellDistance(a, color) < munsellDistance(b, color) ? a : b,
+  );
+
+export const munsellDistance = (a: MunsellHVC, b: MunsellHVC): number =>
+  getDeltaE00(munsellHVCToLAB(a), munsellHVCToLAB(b));
```

Comment:
question: will these be reused? should they go in a color conversions helper file?